spec/integration/cds_importer/entity_mapper_spec.rb

```diff
@@ -336,7 +336,6 @@
     end
 
     it { expect { entity_mapper.import }.to change(AdditionalCode, :count).by(1) }
-    it { expect(entity_mapper.import).to eql('AdditionalCode::Operation' => 1) }
```

Comment:
This is used as part of the 'check sync is still doing something' ? Is this not needed any longer?

Reply:
Not needed anymore, no. The result of each call of each entity mapper would be propagated to the parent via the return but we now use a subscription model to subscribe/publish insertion events